bricks-maven-plugin/src/main/java/com/github/ingogriebsch/bricks/maven/plugin/analyzer/standard/BasicProjectAnalyzer.java

```diff
@@ -0,0 +1,53 @@
+/*-
+ * #%L
+ * Bricks Maven Plugin
+ * %%
+ * Copyright (C) 2018 - 2019 Ingo Griebsch
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.github.ingogriebsch.bricks.maven.plugin.analyzer.standard;
+
+import com.github.ingogriebsch.bricks.maven.plugin.analyzer.AbstractMavenAnalyzer;
+import com.github.ingogriebsch.bricks.maven.plugin.analyzer.AnalysisResult;
+import com.github.ingogriebsch.bricks.model.Component;
+
+import org.apache.maven.project.MavenProject;
+
+public class BasicProjectAnalyzer extends AbstractMavenAnalyzer {
+
+    private static final char DELIMITER = ':';
+
+    @Override
+    protected AnalysisResult augment(Component c) {
+
+        c.setName(project.getName());
+        c.setDescription(project.getDescription());
+
+        // TODO ingo if you had other plans for the id, how about adding a
+        // coordinates attribute to the model?
```

Comment:
Thought about this again and currently stick with an id 'only'. We need to keep in mind that we have components which have nothing to do with the maven like way at all. And I also would like to describe components like 'hand made' docker containers, even if the bricks.json is always written 'by hand' as well. Therefore I would vote for using the artifact id as the component id atm and to rethink the whole way if the parts are a little bit more mature.

Reply:
I'd still add an optional mavenCoordinates attribute, just in case we *can* infer this info.

Reply:
agree?

Reply:
How about adding the maven coordinates to the (not yet available) metadata object?

Reply:
you could move it there, when the time is right.